README.md

```diff
@@ -1 +1,178 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+HTML5  CSS3 JS
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **LOVE**
+- **PASSION**
+- **CODING**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://bleu-yves-sopoude.github.io/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+Create a GitHub acount an clone this repository
+
+### Prerequisites
+
+In order to run this project you need:
+
+a Browser who support Html5 and Css3
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+  cd my-folder
+  git clone https://github.com/Bleu-Yves-Sopoude/mb-portfolio.git
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Bleu Yves Sopoude**
+
+- GitHub: [@Bleu-Yves-Sopoude](https://github.com/Bleu-Yves-Sopoude/mb-portfolio.git)
+
+
+- Linkedin: [@Bleu-Yves-Sopoude](https://www.linkedin.com/in/bleu-yves/)
+
+- Linkedin: [@Bleu-Yves-Sopoude](https://twitter.com/bleuYves)
+
+👤 **Chaw Ma Ma Win**
+
+- GitHub: [@Chaw-Ma-Ma-Win](https://github.com/Chaw-Ma-Ma-Win)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **projects**
+- [ ] **about section**
+- [ ] **formulary**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check my repository in GitHub.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+If you like this project bring me a star in GitHub
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank to my code partners who inspared me to do a better code.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] I would kindly suggest you work on the MIT License link as the link is currently broken and throwing error 404. Having a working link will make your README file more descriptive and professional so please kindly work on it 

MIT 👇 

![mit](https://github.com/Bleu-Yves-Sopoude/AwesomeB-es6/assets/78634841/e13cacd3-7be3-4ff9-ad26-94c1af910a92)